Utils/secrets.py

```diff
@@ -0,0 +1,109 @@
+""" fetch secrets by key from appropriate source according to environment """
+
+from enum import Enum
+import os
+
+
+TEAM_PREFIXES = {"HIVE", "BST"}
+PROJECT_IDS = {
+    "DEV": "amazing-city-422521-t1",
+    "ANA": "ana",
+    "BILL": "bill",
+    "STG": "stg",
+    "PROD": "prod",
+}
+
+
+class Env(Enum):
+    """ values are the key prefixes used in Secret Manager """
+    DEV = "DEV"
+    ANALYTICS = "ANA"
```

Comment:
For Analytics environment, are we thinking read prod and write to dev?

Reply:
It's more about what account is used to connect to which project. I'll think about how to make that clearer in the code. E.g., in `prod`, I assume there will be a main service account (similar to hive robot) and it will connect to `u4u-ds-prod-00` with full r/w access. For `dev-hive`, individual user accounts (e.g. alfred@unrefugees.org) will connect to `u4u-ds-dev-01` with full r/w access. For `analytics`, individual user accounts will connect to `u4u-ds-prod-00` with limited read, and further limited write access.

Furthermore, for the dev and analytics environments, secrets (including GCP user creds) would be read from the local environment. For the rest, they'd be read from Secret Manager.

Does that scheme work for what you have in mind?